src/PointwiseFunctions/AnalyticSolutions/RelativisticEuler/RotatingStar.cpp

```diff
@@ -768,6 +768,16 @@ RotatingStar::variables(
                                 vars->metric_data->alpha, vars->phi);
 }
 
+template <typename DataType>
+auto RotatingStar::variables(
+    const gsl::not_null<IntermediateVariables<DataType>*> vars,
+    const tnsr::I<DataType, 3>& x,
+    tmpl::list<::Tags::dt<gr::Tags::Lapse<DataType>>> /*meta*/) const
+    -> tuples::TaggedTuple<::Tags::dt<gr::Tags::Lapse<DataType>>> {
+  interpolate_deriv_vars_if_necessary(vars);
```

Comment:
So I made this change, but now it's failing to compile in the GitHub tests because `vars` is not used in the function. Should I replace this line with something else so that `vars` is still used?

Reply:
It doesn't look like this was done?

Reply:
Sorry, I guess some weird versioning thing happened that reverted it to what it was before. I'll try again